test/core/tests/secrets_decorator.py

```diff
@@ -26,7 +26,15 @@ def step_all(self):
         import os
         from metaflow import current
 
-        if current.task_id.startswith("control-"):
+        if (
```

Comment:
can you pull this out as a separate PR - this change is likely non-contentious.

Reply:
ya, this is specific to the secrets decorator. Fine leaving it here but totally fine with this change :)

Reply:
I didn't pull it in a separate PR because the change to the decorator also affects the changes in the tests.

Reply:
NVM I pulled it in #1917